rllib/models/modelv2.py

```diff
@@ -266,6 +266,10 @@ def get_view_requirements(
         # Single requirement: Pass current obs as input.
         return {
             SampleBatch.CUR_OBS: ViewRequirement(timesteps=0),
+            SampleBatch.PREV_ACTIONS:
+                ViewRequirement(SampleBatch.ACTIONS, timesteps=-1),
```

Comment:
Why -1 instead of 0?

Reply:
b/c it's the prev-action:
t=0: add obs(0) -> prev-action="np.zeros"-action
t=1: add a(0), r(0), obs(1) -> prev-action=a(t-1=0)
t=2: add a(1), r(1), obs(2) -> prev-action=a(t-1=1)